pull-request-template.md

```diff
@@ -0,0 +1,8 @@
+#### What does this PR do?
+
+
+#### Related JIRA tickets
+
+
+#### Includes new or updated dependencies?
```

Comment:
Adding a section around testing ("how to test this") will be useful.

Reply:
Added